spec/models/institution_spec.rb

```diff
@@ -148,20 +148,11 @@ def create_institution(institution, description)
   end
 
   describe 'class methods and scopes' do
-    context 'with version' do
-      it 'retrieves institutions by a specific version number' do
-        i = create_list :institution, 2, version: 1
-        j = create_list :institution, 2, version: 2
-
-        expect(described_class.version(i.first.version)).to match_array(i.to_a)
-        expect(described_class.version(j.first.version)).to match_array(j.to_a)
-      end
-
-      it 'returns blank if a nil or non-existent version number is supplied' do
```

Comment:
why remove this? should this be reworked to be similar test but for version_id instead?